test/presubmit-backend-test.sh

```diff
@@ -14,14 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# usage: `./hack/run_unit_tests_backend.sh` to run backend unit tests once
-#        `WATCH=true ./hack/run_unit_tests_backend.sh` to watch code changes and auto rerun tests
-# Note: ibazel can be downloaded from https://github.com/bazelbuild/bazel-watcher
-
-COMMAND="bazel"
-if [ -n "$WATCH" ]; then
-  COMMAND="ibazel"
-fi
-$COMMAND --host_jvm_args=-Xmx500m --host_jvm_args=-Xms500m test \
-  --noshow_progress --noshow_loading_progress --define=grpc_no_ares=true \
-  --test_output=all //backend/...
+# The PWD is /home/prow/go/src/github.com/kubeflow/pipelines
+# Navigate to /home/prow/go/
+cd ../../../..
+apt-get update
+apt-get -y upgrade
+wget https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz
```

Comment:
Maybe make it quite?

Reply:
done. btw, sometimes you saw go: downloading.. go: extracting..., which are the output of go mod vendor